specs/electra/beacon-chain.md

```diff
@@ -431,16 +432,18 @@ class BeaconState(Container):
 
 ### Predicates
 
-#### Updated `is_eligible_for_activation_queue`
+#### Updated `is_eligible_for_activation`
 
 ```python
-def is_eligible_for_activation_queue(validator: Validator) -> bool:
+def is_eligible_for_activation(state: BeaconState, validator: Validator) -> bool:
     """
-    Check if ``validator`` is eligible to be placed into the activation queue.
+    Check if ``validator`` is eligible for activation.
     """
     return (
-        validator.activation_eligibility_epoch == FAR_FUTURE_EPOCH
```

Comment:
Correct to assume that validators can never have an `activation_eligibility_epoch > ELECTRA_FORK_EPOCH`? e.g., by depositing during the last slot before electra activates.

Reply:
Those validators are added to the pending deposits queue in [`upgrade_to_electra`](https://github.com/ethereum/consensus-specs/blob/dev/specs/electra/fork.md#upgrading-the-state)